### DISCUSSION TITLE
Converting simple .md to .odt - has something changed?

### DISCUSSION THREAD

**Discussion by philiprhoades** [jgm/pandoc#9444] [General]
For some time I used to be able to convert this file:

```
![](IMG_1294.jpg){ height=90% }

IMG_1294 Notes: 
```
with:
```
pandoc --from=markdown --to=odt $bn.md --reference-doc=custom-reference.odt -o $bn.odt
```
to an ODT file and the image would be maxed to the L / R margins and 90% of the height of the page for the Top / Bottom margins.

I haven't used the process for some time and now the image is too wide for the resulting page.  I have tried messing around with widths instead but the image is not scaling properly anymore . . 

Now using Fedora Linux 39.

Any help appreciated!

Phil.

**Comment by jgm** [comment]
If you want the width not to overflow the page, then it's best to use width=90%.
Pandoc will try to keep the image in proportion, so if you specify height 90% and the image is, say, square, it will overflow width.

**Comment by philiprhoades** [comment]
@jgm ,

I am not expressing myself well - I will try to be clearer by adding diagrams . . 

Previously when I used the method I described above, I would end up with a photo in an A4 page placed nicely between the margins like Page1 - now, the X-axis in the photo idoes not seem to be scaling properly and I am getting pages like Page2 - am I making sense now?

![page1](https://github.com/jgm/pandoc/assets/1153703/6e78a49a-e330-41f9-bf5b-44517bd24d11)


![Page2](https://github.com/jgm/pandoc/assets/1153703/a34b8062-936e-4bac-a052-c6b1a00ceb62)
